tests/main/views/test_services.py

```diff
@@ -463,9 +463,11 @@ def setup(self):
                 **payload
             )
 
-    def _post_service_update(self, service_data, service_id=None):
+    def _post_service_update(self, service_data, service_id=None, user_role=''):
```

Comment:
Nitpicking considering this is a test, but could we have `user_role=None` as the default instead?

Reply:
Sure, if it makes you happy 😄

Reply:
Lovely. Giving the thumbs up now 👍